lib/jekyll_pages_api/page.rb

```diff
@@ -18,23 +21,31 @@ def filterer
     end
 
     def title
-      self.filterer.decode_html(self.page.data['title'] || '')
+      title = self.page.data['title'] if self.page.respond_to?(:data)
+      title = self.page.title if title == nil and self.page.respond_to?(:title)
```

Comment:
Did the code change but to your point, using `is_a?` won't currently work with the tests because Rspec doubles are used, and those don't appear to actually inherit from the original class. I'd experimented with re-opening `Jekyll::Page` et. al. to extend them with new methods, but it failed the tests for the same reason.

So, what I could do is create `DummyPage < Jekyll::Page`, et. al., that prevent the initializers from running, and use those instead of the Rspec-generated doubles. Then I'd prefer to do the class-extension trick rather than `is_a?` or `responds_to?`. Thoughts?